azimuth/plots/dataset_warnings.py

```diff
@@ -437,6 +406,78 @@ def create_histogram_mean_std(
             yanchor="middle",
             showarrow=False,
         )
+    else:
+        y_per_split = {DatasetSplitName.eval: 1.14, DatasetSplitName.train: 1.07}
+        opacity_per_split = {DatasetSplitName.eval: 1, DatasetSplitName.train: 0.5}
+        # sorted because the color overlay looks better when eval is first
+        for split, hist in sorted(hist_normalized.items()):
+            fig.add_bar(
+                x=list(range(1, len(hist) + 1)),
+                y=hist,
+                name=DATASET_SPLIT_PRETTY_NAMES[split],
+                marker=dict(color=DATASET_SPLIT_COLORS[split], opacity=opacity_per_split[split]),
+            )
+
+            fig.add_scatter(
+                x=[np.round(value_per_agg_per_split[split][Agg.mean], 2)],
+                y=[max_y * y_per_split[split]],
+                name=f"{split}_mean_std",
+                error_x=dict(
+                    type="constant", value=np.round(value_per_agg_per_split[split][Agg.std], 2)
+                ),
+                hoverinfo="x",
+                marker=dict(color=DATASET_SPLIT_COLORS[split]),
+            )
+
+        if divergence_per_agg and len(hist_normalized) == 2:
+            fig.add_annotation(
+                x=0,
+                y=max_y * 1.2,
+                text=f"Delta of {divergence_per_agg[Agg.mean]:.2f}±"
+                f"{divergence_per_agg[Agg.std]:.2f} tokens",
+                font=dict(color=Colors.Text),
+                xref="paper",
+                xanchor="left",
+                showarrow=False,
+            )
+
+        # Add thresholds for short and long utterances.
+        max_x = max((len(hist) for hist in hist_normalized.values()), default=0)
+        for x0, x1, pretty_name, xanchor in zip(
+            [0, float(syntax_options.long_utterance_min_word - 0.5)],
+            [float(syntax_options.short_utterance_max_word) + 0.5, max_x + 1],
```

Comment:
Is it necessary to cast to `float()`?
```suggestion
            [0, syntax_options.long_utterance_min_word - 0.5],
            [syntax_options.short_utterance_max_word + 0.5, max_x + 1],
```

If so, is it supposed to be done before subtracting?
```suggestion
            [0, float(syntax_options.long_utterance_min_word) - 0.5],
            [float(syntax_options.short_utterance_max_word) + 0.5, max_x + 1],
```

Reply:
You are right, I don't need to cast to float. I thought I needed it because syntax options are `int`, but it still works. Thanks!